src/chart/sankey/sankeyLayout.js

```diff
@@ -28,6 +28,8 @@ export default function (ecModel, api, payload) {
         var nodeWidth = seriesModel.get('nodeWidth');
         var nodeGap = seriesModel.get('nodeGap');
 
+        var mode = seriesModel.option.mode || 'normal';
```

Comment:
It's suggested to use `seriesModel.get('mode')` instead of `seriesModel.option.mode`.
```suggestion
        var mode = seriesModel.get('mode') || 'normal';
```